crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -397,7 +397,7 @@ impl DWalletMPCManager {
             .map(|party_id| {
                 self.epoch_store()?
                     .committee()
-                    .authority_by_index(*party_id as u32)
+                    .authority_by_index(*party_id as u32 - 1)
```

Comment:
extract this to a function that does the conversion: party_id_to_authority_index.